codersdk/templateversions.go

```diff
@@ -87,7 +87,8 @@ type TemplateVersionVariable struct {
 }
 
 type PatchTemplateVersionRequest struct {
-	Name string `json:"name" validate:"omitempty,template_version_name"`
+	Name    string `json:"name" validate:"omitempty,template_version_name"`
+	Message string `json:"message" validate:"omitempty"`
```

Comment:
Thanks for adding this feature! Should we support setting this to empty string?

User story:

I want to publish a new version where the template has a pre-existing message, but I don't like the message so I simply removed it (empty box). Now I hit publish and the message did not get updated.

We can fix this by making message `Message *string` and defaulting to the existing message, e.g.: 

```go
updateParams.Message = templateVersion.Message
if params.Message != nil {
	updateParams.Message = *params.Message
}
```

Reply:
I think I made that in the patch endpoint but feel free to change anything on the BE